bootstrap/src/main/java/io/airlift/bootstrap/LifeCycleStopException.java

```diff
@@ -0,0 +1,10 @@
+package io.airlift.bootstrap;
+
+public class LifeCycleStopException
+        extends Exception
```

Comment:
Let's make this extend `RuntimeException`. No point in forcing the caller to handle it, since they don't have any reasonable way to do so. I did a quick survey of the ~20 instances in Presto and they all either log or simply rethrow it.